fri/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MIT OR Apache-2.0"
 [dependencies]
 p3-challenger = { path = "../challenger" }
 p3-commit = { path = "../commit" }
+p3-dft = { path = "../dft" }
```

Comment:
I think this can be in dev deps?

Reply:
yep thx

Reply:
for some reason github has been stuck on applying the latest commit to this pr..